index.html

```diff
@@ -41,10 +41,10 @@ <h1 class="logo">My log</h1>
           <section >
               <div class="header-intro">
```

Comment:
- [ ] Again, here am referring to the header. I have noticed that at the zoom percentage of 150%, the social icons go beyond the header boundaries. This is because you have used the fixed height of 552px on the header. Kindly consider making the height flexible such that at all the resizing percentages, the icons don't cross over.
Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/231136035-d014cf6b-fe03-4c33-99a0-b7d7b196f803.png)